staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -328,6 +331,19 @@ func convertToSARExtra(extra map[string][]string) map[string]authorizationv1.Ext
 	return ret
 }
 
+func aggregateErrors(errs []error) error {
```

Comment:
if we need to aggregate an error list, prefer `utilerrors.NewAggregate(errs)` over adhoc aggregation (handles nils, deduplication, nested aggregates, implements errors#Is, etc)